plugins/Kaleidoscope-Colormap-Overlay/src/kaleidoscope/plugin/Colormap-Overlay.h

```diff
@@ -69,6 +82,36 @@ class ColormapOverlay : public kaleidoscope::Plugin {
   void setLEDOverlayColors();
 };
 
+// clang-format off
+
+#define COLORMAP_OVERLAYS_MAP(layers...)                                \
+  namespace kaleidoscope {                                              \
+  namespace plugin {                                                    \
+  const uint8_t overlays_[][kaleidoscope_internal::device.matrix_rows * kaleidoscope_internal::device.matrix_columns] = { \
+      layers                                                            \
+  };                                                                    \
+  ColormapOverlay.configureOverlays(overlays_);                         \
+  } /* plugin */                                                        \
+  } /* kaleidoscope */
+
+#define __IDENTITY__(X) X
+
+#ifdef PER_KEY_DATA_STACKED
+#define COLORMAP_OVERLAY_STACKED(...)                                   \
+  {                                                                     \
+    MAP_LIST(__IDENTITY__, PER_KEY_DATA_STACKED(0, __VA_ARGS__))        \
+  }
+#endif
+
+#ifdef PER_KEY_DATA
+#define COLORMAP_OVERLAY(...)                            \
+  {                                                      \
+    MAP_LIST(__IDENTITY__, PER_KEY_DATA(0, __VA_ARGS__)) \
+  }
+#endif
```

Comment:
This is mostly copy/paste from DefaultColormap. I'm wondering if the namespaces need to be specified like they are 🤔

Reply:
Hm. I guess test it?

Reply:
I haven't felt motivated to pursue this and messing with it could open a can of worms, so I just marked the PR ready to be merged. If I end up pursuing this, I'll do so in a separate PR.